### ISSUE TITLE
Debian - apt repo - package lists or status file could not be parsed or opened

### ISSUE THREAD

**Issue by jamtur01** [timberio/vector#5642]
From [Discord](https://discord.com/channels/742820443487993987/746070591097798688/790431615436193792).

```
So moving from filebeat to vector, I noticed that apt would failt on the few (cough) debian8 systems we have left, due to the vector repo... any chance this could get fixed? What I see is this: 
```

```
 ...
Get:2 https://repositories.timber.io jessie/main amd64 Packages [533 B]
Fetched 5,449 B in 3s (1,443 B/s)
Reading package lists... Error!
E: Problem parsing dependency Depends
E: Error occurred while processing vector (NewVersion2)
E: Problem with MergeList /var/lib/apt/lists/repositories.timber.io_public_vector_deb_debian_dists_jessie_main_binary-amd64_Packages
E: The package lists or status file could not be parsed or opened.
```

**Comment by jamtur01** [comment]
This is likely an empty:

```
Depends:
```

line - which I think Debian 8 and older dpkg chokes on.

**Comment by kbarmen** [comment]
Content of my /etc/apt/sources.lists.d/vector.list is this:

`deb https://repositories.timber.io/public/vector/deb/debian jessie main`

**Comment by jamtur01** [comment]
This should be fixed now for Nightly onwards builds. Please re-open if you still have issues.

**Comment by kowoba** [comment]
Just checking back on this now, and the problem is stille the same as far as I can tell.

"apt-get update" produces a "/var/lib/apt/lists/repositories.timber.io_public_vector_deb_debian_dists_jessie_main_binary-amd64_Packages" that has empty "Depends:" entries.

No sure what you mean with "Nightly onwards builds".

**Comment by kbarmen** [comment]
Just checking back on this now, and the problem is stille the same as far as I can tell.

"apt-get update" produces a "/var/lib/apt/lists/repositories.timber.io_public_vector_deb_debian_dists_jessie_main_binary-amd64_Packages" that has empty "Depends:" entries.

No sure what you mean with "Nightly onwards builds".

**Comment by jamtur01** [comment]
Can you confirm what Debian release you are on?

**Comment by kbarmen** [comment]
8.11 aka jessie, amd64

[code]
Distributor ID: Debian
Description:    Debian GNU/Linux 8.11 (jessie)
Release:        8.11
Codename:       jessie
[/code]

**Comment by jamtur01** [comment]
c.f. https://github.com/mmstick/cargo-deb/issues/161

**Comment by kbarmen** [comment]
The problem is still there, I don't grasp why you keep closing the ticket.

**Comment by jszwedko** [comment]
@kbarmen The fix should go out with the next release, but is available in the nightly builds now.

I'll bring up whether we could backport this to the latest release.

**Comment by jszwedko** [comment]
Actually, this is a lie. I still see it with the nightlies. I'll reopen this.

**Comment by jamtur01** [comment]
@kbarmen Because I keep thinking I fixed it. :) 

I can see now that the problem is in the 0.11 packages in the same manifest. It's fixed for 0.12 and I only tested that.

```
root@5b32dfb7a25f:/var/lib/apt/lists# cat repositories.timber.io_public_vector-nightly_deb_ubuntu_dists_xenial_main_binary-amd64_Packages
Package: vector
Version: 0.11.0
Architecture: amd64
Homepage: https://vector.dev
Priority: optional
Standards-Version: 3.9.4
Maintainer: Vector Contributors <vector@timber.io>
Installed-Size: 116805
Depends:
Description: A lightweight and ultra-fast tool for building observability pipelines
 .
Filename: pool/any-version/main/v/ve/vector-0.11.0-amd64.deb
Size: 24934430
MD5Sum: e2f68777c12c0516da7d9601612ebfce
SHA1: 404baddaf0faeb0b9de57df09cf57a4758f8ae2d
SHA256: 0176682356048d61b3a13ce8abb5f623716752181c2e7400a0905ea3b2963035

Package: vector
Version: 0.12.0
Architecture: amd64
Homepage: https://vector.dev
Priority: optional
Standards-Version: 3.9.4
Maintainer: Vector Contributors <vector@timber.io>
Installed-Size: 63173
Depends: libc6 (>= 2.17)
Description: A lightweight and ultra-fast tool for building observability pipelines
 .
Filename: pool/any-version/main/v/ve/vector-0.12.0-amd64.deb
Size: 15430524
MD5Sum: 9d32dd93ec178b1bd1d2ba79172fc47b
SHA1: a6bf3c0b62301162b14bfffc19b5a8c909f40d1a
SHA256: 5b5f7abc85d25e336519a4de34e3129bbef51f6ad55ae92ecf0aadb693ff6603
```

Fixing this means going back to rebuild the Debian 0.11 packages to add a hard-coded `Depends` in them.

**Comment by amexboy** [comment]
> Fixing this means going back to rebuild the Debian 0.11 packages to add a hard-coded Depends in them.

Does this still mean we have to wait for the 0.12 release? Or would it be fixed before?

**Comment by jszwedko** [comment]
We released 0.12.0, but this is still an issue since the old packages have the `Depends` line. We'll need to backport the fix. I'll look into this.

**Comment by jszwedko** [comment]
Apologies for the long delay in this one @amexboy . I was able to backport the fix to 0.11.0 and 0.11.1 so the apt manifest no longer fails to parse on Debian 8 (I verified using Docker).

I still need to update the nightly repo to fix the 0.11 version over there. I'll aim to do that tomorrow.

**Comment by jszwedko** [comment]
I was able to republish the nightly packages with the fix as well so I think we are clear of this issue now. Thanks for raising it up @amexboy . Let me know if you still see any issues.
